init/minimal_eessi_env

```diff
@@ -8,7 +8,17 @@ EESSI_INIT_DIR_PATH=$(dirname $(readlink -f $BASH_SOURCE))
 #   script takes *_OVERRIDEs into account
 source ${EESSI_INIT_DIR_PATH}/eessi_defaults
 
-export EESSI_PREFIX=$EESSI_CVMFS_REPO/versions/$EESSI_VERSION
+# If $EESSI_DEV_PROJECT is defined, then we are building for dev.eessi.io
+# and add a subdirectory to $EESSI_PREFIX
+if [ -z ${EESSI_DEV_PROJECT} ]; then
+    export EESSI_PREFIX=$EESSI_CVMFS_REPO/versions/$EESSI_VERSION
+else
+    echo "\$EESSI_DEV_PROJECT: ${EESSI_DEV_PROJECT}; building for dev.eessi.io"
+    export EESSI_PREFIX=$EESSI_CVMFS_REPO_OVERRIDE/versions/$EESSI_VERSION/$EESSI_DEV_PROJECT
+    # Set EESSI_PREFIX to the dev project path for EESSI-extend
+    export EESSI_CVMFS_INSTALL=$EESSI_PREFIX
+fi
```

Comment:
Ditto here, you could just tweak the definition of `EESSI_CVMFS_REPO_OVERRIDE`

Reply:
Agreed, but I think we need to set `$EESSI_CVMFS_REPO_OVERRIDE` before we call `minimal_eessi_env`, in `bot/build.sh`(?). I'm trying this now in acb896a163118d103e62e65566e052ce05c8ccc6